draft-pignataro-eimpact-icmp.xml

```diff
@@ -553,6 +624,45 @@
       </section>
     </section>
 
+    <section anchor="subobject-formats" title="Sub-Object Formats">
+      <t>
+        As discussed in <xref target="ctype" />, apart from the 32-bit "short" 
+        sub-object, we have also defined a 64-bit "wide" format for Node and 
+        Component throughput Sub-Objects.
+      </t>
+      
+      <t>
+        The 64-bit format was defined keeping the present trends of the industry in
+        mind. With the industry shifting towards high density ports/components and 
+        nodes, organizations now have components/interfaces/line cards with 10/40/100Gbps 
+        or even 400Gbps throughput and nodes with a combined throughput of greater than
+        10Tbps. Now this cannot be represented in a field wihich is only 32-bit long. 
+        Because a 32-bit field can only represent a max throughput of 4Gbps (2 ^ 32 BPS).
+        So, to address this, we chose to define another format, a wider, 64-bit Sub-Object
+        format.
+      </t>
+      
+      <t>
+        Now that we have put forward these two format options, here's how a device MUST
+        choose amongst the available format options. It is recommended that the device
+        chooses a format based on the size of the value that it wants to append. For 
+        example, If the component/node throughput is more less than 4Gbps, the device 
+        MUST append the "short" 32-bit Sub-Object. And when the component/node throughput
```

Comment:
```suggestion
        would append the "short" 32-bit Sub-Object. When the component/node throughput
```

Let's avoid RFC 2119 style words in an example. And: I don't see why we need to be so strict.